WPFHospitalEditor/EquipmentAndMedicationWindow.xaml.cs

```diff
@@ -9,17 +12,45 @@ namespace WPFHospitalEditor
     /// </summary>
     public partial class EquipmentAndMedicationWindow : Window
     {
-        public EquipmentAndMedicationWindow(String[] contentRows)
+        private IEquipmentTypeServerController equipmentTypeServerController = new EquipmentTypeServerController();
+        int mapObjectId;
+        HospitalMap hospitalMap;
+        Role user;
```

Comment:
Don't need field user because it's static field in HospitalMap. And, beside that, you use it only in constructor.

Reply:
Yes, my bad..

Reply:
Fixed.